source/includes/_processing.md

```diff
@@ -0,0 +1,314 @@
+
+## Update a processor
+
+This endpoint is meant to update a processor in place within the processing stack of a dataset.
+Explanation of the thing
+
+> Definition
+
+```
+PUT https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_abcdef
+    -u username:password \
+    -d '{"name":"regexp_replace", "args": {"field": "text_field", "regexp": ".*", "new": "Bonjour Monde"}'
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_abcdef", 
+    "name":"regexp_replace",
+    "args": {
+        "field": "text_field",
+        "regexp": ".*",
+        "new": "Hello World"
+    }
+}
+```
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+`name` <br> *string* | name of the processor
+`args` <br> *object* | processor-dependent parameters
+
+### Returns
+The newly updated [processor object](#the-processor-object).
+
+## Delete a processor
+
+This endpoint is meant to delete a processor from the processing stack of a dataset
+
+> Definition
+
+```HTTP
+DELETE https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XDELETE https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_newnew
+    -u username:password
+```
+
+### Returns
+
+On successful deletion, the endpoint returns a HTTP 204 without any content.
+
+
+## Retrieve possible processor parameters from processor name
+
+For some processors and some processors parameters, the OpenDataSoft platform has a way of determining a set of possible values. It can for instance figure out the list of fields the processor could operate on.
+
+This endpoint is meant to expose this feature, by taking a processor name and returning the set of possible values for each parameter.
+
+> Definition
+
+```HTTP
+POST https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/guess_processor_params
+```
+
+> Example request
+
+```HTTP
+curl -XPOST https://yourdomain.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/guess_processor_params
+    -u username:password \
+    -d '{"name": "expand_json_array"}'
+}
+```
+
+> Example response
+
+```json
+{
+    "json_array_field": {
+        "text_field": {
+            "annotations": [],
+            "description": null,
+            "label": "text_field",
+            "name": "text_field",
+            "original_name": "text_field",
+            "type": "text"
+        }
+    }
+}
+```
+
+The endpoint takes the name of a processor, and tries to find the list of values that would satisfy the parameters where possible as if the processor was appended to the end of the processing stack. The parameters are returned in a object structure, with their specifications expanded. Some parameters cannot be guessed.
```

Comment:
This is quite hard to understand too, maybe you could rephrase a bit?

Reply:
:+1: